.github/workflows/pr_comment_bot.yml

```diff
@@ -135,9 +135,7 @@ jobs:
     needs: [pr_comment]
     if: |
       needs.pr_comment.outputs.command == 'run-tests' ||
-      needs.pr_comment.outputs.command == 'run-tests-extended' ||
-      needs.pr_comment.outputs.command == 'run-tests-extended-aad' ||
-      needs.pr_comment.outputs.command == 'run-tests-shared-services'
+      needs.pr_comment.outputs.command == 'run-tests-extended'
```

Comment:
Shouldn't the command be still available for cases where developers know what they want to test?
For example, if I change a shared service being tested I will call the run-shared-services without any of the workspace level stuff?

Reply:
sort of agree - however now that workspaces + services can update those shared services, and do, there's also a case to be made that we should be running all the tests as we need to test the pipeline update from a workspace -> shared service?

Reply:
> Shouldn't the command be still available for cases where developers know what they want to test?

I think a more future-proof is would be to run a workflow with a custom selector. This way we can easily extend selectors without having to update PR bot each time. 

>  there's also a case to be made that we should be running all the tests as we need to test the pipeline update from a workspace -> shared service?

Well other bundles only really update firewall, and that will be tested by deploying said bundles. I don't think we would need to test shared services each time